ztp/gitops-subscriptions/argocd/example/acmpolicygenerator/acm-common-ranGen.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: PolicyGenerator
+metadata:
+  name: common-latest
+placementBindingDefaults:
+  name: common-placement-binding
+policyDefaults:
+  # categories: []
+  #controls:
+  #  - PR.DS-1 Data-at-rest
+  namespace: ztp-common
+  # Use an existing placement rule so that placement bindings can be consolidated
+  placement:
+    labelSelector:
+      common: "true"
+      du-profile: "latest"
+  remediationAction: inform
+  severity: low
+  # standards: []
+  namespaceSelector:
+    exclude:
+      - kube-*
+    include:
+      - '*'
+  evaluationInterval:
+    compliant: 10m
+    noncompliant: 10s
+policies:
+  # Create operators policies that will be installed in all clusters
+- name: common-subscriptions-policy-latest
+  policyAnnotations:
+    ran.openshift.io/ztp-deploy-wave: "2"
+  manifests:
+    - path: source-crs/SriovSubscriptionNS.yaml
+    - path: source-crs/PtpSubscriptionNS.yaml
+    - path: source-crs/ClusterLogNS.yaml
+    - path: source-crs/StorageNS.yaml
+    - path: source-crs/SriovSubscription.yaml
+    - path: source-crs/SriovSubscriptionOperGroup.yaml
```

Comment:
Switch these two so that the opergroup is created before the subscription (sub needs the opergroup)